gluonnlp/model/language_model.py

```diff
@@ -153,8 +106,7 @@ def forward(self, inputs, begin_state=None): # pylint: disable=arguments-differ
             out = self.decoder(encoded)
         return out, out_states
```

Comment:
maybe we can use the forward function from parent class, such as 
```python
outputs = super(AWDRNN, self).forward(xxxxx)
return outputs[0], outputs[1]
```

Reply:
The super's forward will preserve the encoder outputs and dropped encoder outputs in memory. I  prefer to keep it the same.